resources/qml/WelcomePages/WelcomeContent.qml

```diff
@@ -7,56 +7,76 @@ import QtQuick.Controls 2.3
 import UM 1.3 as UM
 import Cura 1.1 as Cura
 
-
 //
 // This component contains the content for the "Welcome" page of the welcome on-boarding process.
 //
 Item
 {
     UM.I18nCatalog { id: catalog; name: "cura" }
 
-    Column  // Arrange the items vertically and put everything in the center
+    // Arrange the items vertically and put everything in the center
+    Column
     {
-        anchors.centerIn: parent
-        width: parent.width
-        spacing: 2 * UM.Theme.getSize("wide_margin").height
-
-        Label
-        {
-            id: titleLabel
-            anchors.horizontalCenter: parent.horizontalCenter
-            horizontalAlignment: Text.AlignHCenter
-            text: catalog.i18nc("@label", "Welcome to Ultimaker Cura")
-            color: UM.Theme.getColor("primary_button")
-            font: UM.Theme.getFont("huge")
-            renderType: Text.NativeRendering
-        }
+        anchors.horizontalCenter: parent.horizontalCenter
+        anchors.verticalCenter: parent.verticalCenter
+        spacing: UM.Theme.getSize("thick_margin").height
 
         Image
         {
             id: curaImage
             anchors.horizontalCenter: parent.horizontalCenter
             source: UM.Theme.getImage("first_run_welcome_cura")
+            scale: 0.666
         }
 
-        Label
+        Column
         {
-            id: textLabel
             anchors.horizontalCenter: parent.horizontalCenter
-            horizontalAlignment: Text.AlignHCenter
-            text: catalog.i18nc("@text", "Please follow these steps to set up\nUltimaker Cura. This will only take a few moments.")
-            font: UM.Theme.getFont("medium")
-            color: UM.Theme.getColor("text")
-            renderType: Text.NativeRendering
+            spacing: UM.Theme.getSize("thick_margin").height
+
+            Label
+            {
+                id: titleLabel
+                anchors.horizontalCenter: parent.horizontalCenter
+                horizontalAlignment: Text.AlignHCenter
+                text: catalog.i18nc("@label", "Welcome to Ultimaker Cura")
+                color: UM.Theme.getColor("primary_button")
+                font: UM.Theme.getFont("huge_bold")
+                renderType: Text.NativeRendering
+            }
+
+            Label
+            {
+                id: textLabel
+                anchors.horizontalCenter: parent.horizontalCenter
+                horizontalAlignment: Text.AlignHCenter
+                text: catalog.i18nc("@text", "Please follow these steps to set up Ultimaker Cura.\nThis will only take a few moments.")
```

Comment:
(In this case the text contains a newline, but it would not be a problem for the translators since each sentence is on its own line. They'll just do the same. You improved the situation here. Just be aware that the lines may not be equally long in a different language so if space is tight this might become a problem.)